Use the default option as in catmandu #333 (#356)
metafacture/metafacture-fix: README.md

Options:

> - `"default"`: Default value to use for unknown values. The `"default"`-key needs to be in quotation marks since it is a reserved word in Java.  (Default: Old value)

Reviewer: Please don't mix names and syntax, the option name is "default". The fact that it (currently) has to be written as `"default"` in the Fix should not matter here. I'd be willing to accept the note, but it should be reworded (also, extra whitespace):

```suggestion
- `default`: Default value to use for unknown values. The option needs to be written in quotation marks because it is a reserved word in Java. (Default: Old value)
```

Author: Done thanks. We could use the wording in #365  too.

lookup("path.to.field", "rdf-map")

# with default value

> lookup("path.to.field", "map-name", default: "NA")

Reviewer: The option needs to be written in quotation marks because it is a reserved word in Java.

Author: ```suggestion
lookup("path.to.field", "map-name", "default": "NA")
```

Author: Done. Thanks
